### PR TITLE
Prettier config and formatting for ruby

### PR DESCRIPTION
**Pull request by taylorkearns**
Updates prettier ruby packages and formats existing ruby.

### CODE REVIEW

File: app/models/asset.rb [hunk 1]
```diff
@@ -2,14 +2,14 @@ class Asset < ApplicationRecord
...
-    chair: "chair",
-  }
+         laptop: "laptop",
+         display: "display",
+         keyboard: "keyboard",
+         mouse: "mouse",
+         power_supply: "power supply",
+         desk: "desk",
+         chair: "chair"
+       }
```
Comment (alxjrvs) [inline app/models/asset.rb:12]: I do not like this change. Why so many spaces (tabs)?

**Comment by alxjrvs** [comment]
Also, I see what is happening with the `pronto` failure here. 

Long story short: If the action doesn't *already* support a plugin, we cannot use that plugin with `rubocop`. 

I have had success opening a pull request into that action and manually adding gems to their gemfile. 

This is a restriction of the action, and the last time we ran into this issue [we solved it with a PR](https://github.com/HeRoMo/pronto-action/pull/160). Could fork it and make a PR (or just fork it and use the fork)

File: app/models/asset.rb (hunk 1 above)
Comment (taylorkearns) [reply app/models/asset.rb:12]: Yea good question. I suspect it is aligning two spaces from the end of `asset_type`? I'll see about turning off or adjusting that cop.

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 2]
```diff
@@ -7,18 +7,22 @@ def google_oauth2
       @user = User.from_omniauth(request.env["omniauth.auth"])
 
       if @user.persisted?
-        flash[:notice] = I18n.t "devise.omniauth_callbacks.success", kind: "Google"
+        flash[:notice] = I18n.t "devise.omniauth_callbacks.success",
+               kind: "Google"
```
Comment (taylorkearns) [inline app/controllers/users/omniauth_callbacks_controller.rb:11]: This indentation is peculiar as well.

**Comment by alxjrvs** [comment]
@taylorkearns I rebased and updated the pronto action here, but upon closer inspection my congratulations in the commit was presumptuous.

**Comment by taylorkearns** [comment]
@alxjrvs I'll need to look closer at what's going on, not sure why that didn't take. FWIW the maintainer rejected my PR (appropriately) because prettier isn't a pronto add on.

**Comment by alxjrvs** [comment]
@taylorkearns interesting. Do we want this instead? https://github.com/oozou/pronto-prettier

**Comment by taylorkearns** [comment]
@alxjrvs maybe? I looked at this previously but it hasn't been touched in 4 years.

**Pull request closed without merging**